docs/src/man/basics.md

```diff
@@ -52,6 +52,37 @@ The most fundamental type provided by DataFrames.jl is `DataFrame`, where
 typically each row is interpreted as an observation and each column as a
 feature.
 
+!!! note
```

Comment:
@timholy - could you please quickly check if this is a correct description?

In particular I had to use `force=true` option as otherwise `set_preferences!` was complaining when I wanted to switch on/off precompilation of DataFrames.jl.

Now the effect of disabling precompilation for DataFrames.jl is:
* first load time (with precompilation): 5.5 seconds
* next load times 0.8 seconds